### PR TITLE
Kw user logs out

### PR DESCRIPTION
**Pull request by townie**
@HeroicEric  here are the changes!

### CODE REVIEW

File: spec/features/create_a_new_mugshot_spec.rb [hunk 1]
```diff
@@ -20,7 +20,10 @@
     fill_in 'First Name', with: 'Justin'
     fill_in 'Last Name', with: 'Bieber'
     fill_in 'Description', with: 'This guy should not be drunk driving'
-    fill_in 'Approximate Date', with: '10/3/13'
+    #select "Feature Request", from: "Category"
+    select "2014" && "April" && '11', from: "Approximate Date"
+    # select , from: "Approximate Date"
+    # select "1", from: "Approximate Date"
```
Comment (HeroicEric) [inline spec/features/create_a_new_mugshot_spec.rb]: Should not commit code that is commented out.

File: spec/features/user_logs_out_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+feature 'user logs out', %q{
+  As a registered user
+  I can logout of the system
+  So that I am not logged in anymore
+} do
+  # *If I am logged in, from any page i see a logout button
+  # *When I click logout I am no longer logged in
+  # *I am redirected back to the home page
+
+
```
Comment (HeroicEric) [inline spec/features/user_logs_out_spec.rb]: Extra newlines

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -13,6 +13,7 @@
 
   <body>
 
+
```
Comment (HeroicEric) [inline app/views/layouts/application.html.erb]: Why was this added?

File: spec/features/create_a_new_mugshot_spec.rb (hunk 1 above)
Comment (townie) [inline spec/features/create_a_new_mugshot_spec.rb]: How do I blame @DMJr

**Comment by townie** [comment]
@HeroicEric fixed conflicts

**Comment by townie** [comment]
@HeroicEric merge please!

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -44,6 +44,7 @@
     </div>
   </div>
 
+
```
Comment (HeroicEric) [inline app/views/layouts/application.html.erb]: why?

File: config/routes.rb [hunk 5]
```diff
@@ -1,5 +1,6 @@
 CelebrityMugshots::Application.routes.draw do
 
+
```
Comment (HeroicEric) [inline config/routes.rb]: why?

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (townie) [inline app/views/layouts/application.html.erb]: to get under ur skin

**Comment by HeroicEric** [comment]
@townie doesn't merge cleanly with master

**Comment by townie** [comment]
@HeroicEric cleanly merging
